SU-HSDO/suhumsci
# SHS-5696: UI : Entering text into the Block quote space appears italicized

# READY FOR REVIEW

## Summary
Update the font style of blockquotes in WYSIWYG.

## Need Review By (Date)
07/26

## Urgency
medium

## Steps to Test
1. On Colorful and Traditional
2. Create a new Flexible page 
3. In the text area, add a `Blockquote`
4. Enter text into the Block quote area, check the text is not longer italicized when editing the text.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207890941895360
